EpikCord/client/http.py

```diff
@@ -24,6 +24,7 @@
     json_serialize,
     log_request,
 )
+from ..utils.enums import HTTPCodes
```

Comment:
You don't need the `.enums` part, just import from `utils`.

Reply:
> You don't need the `.enums` part, just import from `utils`.

```suggestion
from ..utils import HTTPCodes
```

Reply:
![image](https://user-images.githubusercontent.com/53050011/215899609-01cef8f0-20c2-488b-b7ca-5e88391f3c51.png)